stock_cubiscan/models/stock.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class StockWarehouse(models.Model):
+    _inherit = "stock.warehouse"
+
+    cubiscan_device_ids = fields.One2many(
+        "cubiscan.device", "warehouse_id", string="Cubiscan Devices"
+    )
+
+
+class ProductPackaging(models.Model):
+    _inherit = "product.packaging"
+    # FIXME: move this in product_packaging_type
```

Comment:
@guewen still the case?

Reply:
yes, it should be in this module IMO https://github.com/OCA/product-attribute/pull/536

Reply:
can you update the comment w/ a link to https://github.com/OCA/product-attribute/tree/13.0/product_packaging_type ?